apps/judicial-system/backend/src/app/modules/case/filters/case.filter.ts

```diff
@@ -130,9 +130,18 @@ function canAppealsCourtUserAccessCase(theCase: Case): boolean {
   // Check appeal state access
   if (
     !theCase.appealState ||
-    ![CaseAppealState.RECEIVED, CaseAppealState.COMPLETED].includes(
-      theCase.appealState,
-    )
+    ![
+      CaseAppealState.RECEIVED,
+      CaseAppealState.COMPLETED,
+      CaseAppealState.WITHDRAWN,
+    ].includes(theCase.appealState)
+  ) {
+    return false
+  }
+
+  if (
+    theCase.appealState === CaseAppealState.WITHDRAWN &&
+    !theCase.appealReceivedByCourtDate
```

Comment:
Please consider updating the appropriate test in `appealsCourtUserFilter.spec.ts`.

Reply:
Done